cleanlab/models/keras.py

```diff
@@ -75,10 +75,12 @@ def __init__(
         compile_kwargs: dict = {
             "loss": tf.keras.losses.SparseCategoricalCrossentropy(from_logits=True)
         },
+        params: dict = {},
```

Comment:
Safer to use `None` as a default.
```suggestion
        params: Optional[dict] = None,
```

Then set the attribute to an empty dictionary if the argument is `None`

Reply:
What is the reason that `None` is safer than an empty dictionary? (are there risks to using the empty dict as default?)

Reply:
It's just risky to use an empty dictionary as a default argument if we end up referring to the argument elsewhere and accidentally modify it.

```python
def add_to_empty(empty_dict: dict = {}):
    assert not empty_dict, f"empty_dict must be empty, but it is not: \n{empty_dict}"
    empty_dict["key"] = "value"

if __name__ == "__main__":
    print("First call:")
    add_to_empty() 
    print("Second call:")
    add_to_empty()
```

Running the above prints out something like:

```
First call:
Second call:
...
AssertionError: empty_dict must be empty, but it is not: 
{'key': 'value'}